docs/DeveloperGuide.md

```diff
@@ -539,6 +556,7 @@ testers are expected to do more *exploratory* testing.
       Expected: 
       First client is unarchived, and is no longer shown in the archive. Details of the unarchived client shown in the status message.
       Upon listing all active clients (see [List Active Clients](#listing-all-active-clients)), the unarchived client can be seen.
+      The color of the priority indicator is lightened to show that the client has been unarchived.
```

Comment:
Maybe we can have a diagram here of both lightened and darkened? Can ask them to refer to the previous diagram to avoid duplication

Reply:
yupps

Reply:
Approving this first but diagram is pending